frontend/scripts/check-boundary-keys-validity.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * This script checks that the keys listed in layers.json
+ * actually exist in the corresponding boundary files.
+ */
+const fs = require('fs');
+const path = require('path');
+
+let missingKeys = false;
+
+function checkKeysExistence(layer, filePath) {
+  const fileContent = fs.readFileSync(filePath, 'utf-8');
+
+  // Check if the keys exist in the JSON file
+  const keysToCheck = [
+    ...layer.admin_level_codes,
+    ...layer.admin_level_names,
+    ...layer.admin_level_local_names,
+  ];
+
+  keysToCheck.forEach(key => {
+    if (!fileContent.includes(key)) {
```

Comment:
If we're very rigorous, this only checks that the key string appears in the content of the file, not that it's actually a JSON key, nor that it's in the right place. I guess a more thorough check would be provided by a json schema or equivalent, but that's probably a bit more work to implement?

Reply:
I am testing `"${key}":` to make this a bit more robust